kymatio/scattering1d/backend/torch_skcuda_backend.py

```diff
@@ -62,29 +62,29 @@ def __call__(self, x):
         if not x.is_cuda and self.backend=='skcuda':
             raise TypeError('Use the torch backend (without skcuda) for CPU tensors.')
 
-        out = x.new(x.shape)
+        out = x.new(x.shape[:-1] + (1,))
```

Comment:
for my understanding: what is `x.shape` expected to be here? Is it expected to be complex, i.e. with a 2 at the end?

I'd prefer `torch.empty_like(x[..., :1])`

Reply:
its a tuple with a 2 at the end. I want every-part of the tuple but the complex dim. I add the (1,) to get a real tuple.